packages/web-react/package.json

```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "@react-hook/resize-observer": "^1.2.6",
     "classnames": "^2.3.1",
+    "react-html-parser": "^2.0.2",
```

Comment:
I am not an expert here, I just noticed that the library wasn't updated for 6 years. Maybe it doesn't need it, just FYI :)

Reply:
You are right. I have found something newer: https://www.npmjs.com/package/html-react-parser